CHANGELOG.md

```diff
@@ -9,6 +9,22 @@
 * **[Fix]** Fix exception handling when the app tries to access a locked database.
 * **[Fix]** Fix getting device information when the application runs in partial trust mode.
 
+#### iOS
+
+* **[Fix]** Fix compatibility with Xcode 12 beta when integrating SDK from sources.
```

Comment:
This doesn't affect .NET, we need to remove this line

Reply:
thanks, I modified this.